### PR TITLE
Document a prerequisite: generating an SSH key and adding it to GitHub

### PR DESCRIPTION
**Pull request by issyl0**
- I've been going through the instructions word for word, and hit this
  problem with "please make sure you have the correct access rights"
  when cloning the boxen repo as instructed further down in the
  README.

### CODE REVIEW

**Comment by bradleywright** [comment]
Would be nice to link to the [official GitHub guidance](https://help.github.com/articles/generating-ssh-keys/) as that'll be more likely to be kept up to date (and it's more in-depth as well).

**Comment by issyl0** [comment]
Aha! Better?

File: README.md [hunk 1]
```diff
@@ -23,7 +24,12 @@
     echo '[ -f /opt/boxen/env.sh ] && source /opt/boxen/env.sh' >> ~/.zshrc
 
 It should run successfully, and should tell you to source a shell script
-in your environment.
+in your environment. If it does not run successfully, and gives you
+errors about unable to resolve `github.gds`, make sure you're on the
+Bardeen wireless network and can access `github.gds` in a web browser.
```
Comment (bradleywright) [inline README.md]: Not just Bardeen - probably better to say "any of the internal staff wifi networks".

**Comment by bradleywright** [comment]
Other than my last petty complaint, :+1:

**Comment by bradleywright** [comment]
Also thanks for making this easier for future people.

File: README.md (hunk 1 above)
Comment (issyl0) [inline README.md]: OK. Should we specify in that case "not Shockley"?

Comment (bradleywright) [inline README.md]: I don't think we should put corporate wifi names on documentation in public at all - as the past 6 months shows they're likely to change and then this is out of date.

Comment (issyl0) [inline README.md]: OK, removed.

**Pull request merged**